### PR TITLE
Update to DDF 2.25.1

### PR DESCRIPTION
**Pull request by jlcsmith**
#### What does this PR do?
Updates to DDF 2.25.1
Removes experimental profile

#### Who is reviewing it? 
@rzwiefel 
@bdeining 

#### Choose 2 committers to review/merge the PR.
(please choose ONLY two committers from below, delete the rest)

@rzwiefel 
@bdeining 

#### What are the relevant tickets?

[CAL-xxx](https://codice.atlassian.net/browse/CAL-xxx)
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

**Comment by jlcsmith** [comment]
build now

**Comment by jlcsmith** [comment]
build now

File: distribution/install-profiles/src/main/resources/features.xml [hunk 1]
```diff
@@ -29,31 +29,9 @@
         <feature>catalog-solr-app</feature>
 
         <!-- Alliance Apps -->
+        <feature>video-app</feature>
```
Comment (shaundmorris) [inline distribution/install-profiles/src/main/resources/features.xml:32]: ❓ Intersting so video-app was only in the profile-alliance-experimental profile?

Comment (jlcsmith) [reply distribution/install-profiles/src/main/resources/features.xml:32]: Yes other than resource mgmt and GWC.  Video app should have been moved out of experimental long ago given its  community use.

**Pull request merged**